src/zimdump.cpp

```diff
@@ -395,7 +395,7 @@ int subcmdDump(ZimDumper &app,  std::map<std::string, docopt::value> &args)
     std::string directory = args["--dir"].asString();
 
     if (directory.empty()) {
-        throw std::runtime_error("Directory cannot be empty.");
+        //throw std::runtime_error("Directory cannot be empty.");
```

Comment:
Same here. This test is used to not overwrite a existing content in the output directory.
We don't want to remove it.